core/types/src/persistence.ts

```diff
@@ -0,0 +1,33 @@
+export type BaselineCommitment = {
```

Comment:
We should add a `sender` to this commitment type also, indicating the address of the participant who created the commitment... i.e., this is the party that will ultimately call `verifyAndPush`.

Reply:
Let's add a `metadata` field here too.